router/transformer/transformer_test.go

```diff
@@ -985,8 +985,8 @@ var oauthv2ProxyTestCases = []oauthv2ProxyTcs{
 			ProxyRequestResponseBody: `{"message": "some message that we got from transformer","authErrorCategory":"AUTH_STATUS_INACTIVE","response":[{"statusCode":403,"error":"permission is not present","metadata":{"workspaceId":"wsp","destinationId":"d1","jobId":2}},{"statusCode":403,"error":"permission is not present","metadata":{"workspaceId":"wsp","destinationId":"d1","jobId":1}}]}`,
 			ProxyRequestStatusCode:   400,
 			RespStatusCodes: map[int64]int{
-				1: 403,
-				2: 403,
+				1: 400,
+				2: 400,
```

Comment:
Similar for below as well.
```suggestion
				1: http.StatusBadRequest,
				2: http.StatusBadRequest,
```

Reply:
Done